spp_land_record/__manifest__.py

```diff
@@ -16,6 +16,9 @@
         "spp_base_gis",
         "g2p_registry_base",
     ],
+    "excludes": [
+        "spp_base",
+    ],
```

Comment:
@gonzalesedwin1123 same comment as above.

Reply:
@reichie020212 Is this not for farmer registry only?

Reply:
@gonzalesedwin1123 currently only the farmer registry is using this module but this module is generic and can be used by other modules as well.